mlflow/recipes/steps/train.py

```diff
@@ -327,40 +335,69 @@ def my_warn(*args, **kwargs):
                 mlflow.sklearn.log_model(
                     transformer, "transform/transformer", code_paths=self.code_paths
                 )
-                model = make_pipeline(transformer, estimator)
-                if self.predict_proba:
-                    model_schema = infer_signature(
-                        raw_X_train, model.predict_proba(raw_X_train.copy())
-                    )
-                    model_info = mlflow.sklearn.log_model(
-                        model,
-                        f"{self.name}/model",
-                        signature=model_schema,
-                        code_paths=self.code_paths,
-                        pyfunc_predict_fn="predict_proba",
-                    )
-                else:
-                    model_schema = infer_signature(raw_X_train, model.predict(raw_X_train.copy()))
-                    model_info = mlflow.sklearn.log_model(
-                        model,
-                        f"{self.name}/model",
-                        signature=model_schema,
-                        code_paths=self.code_paths,
-                    )
-                output_model_path = get_step_output_path(
+
+                trained_pipeline = make_pipeline(transformer, estimator)
+                # Creating a wrapped recipe model for the pipeline so users can achieve both
+                # predict and predict_proba(for a classification problem) correctly.
```

Comment:
The wrapped model still only exposes one API (`predict()`), right? Can we rephrase this to indicate that the wrapped model is created so that `predict()` can output class predictions and probabilities at the same time?